src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -45,7 +45,9 @@ export class DotEditPageNavComponent implements OnInit {
             }
         ];
 
-        result.push(this.getTemplateNavItem(dotRenderedPage));
+        if (dotRenderedPage.template.drawed) {
```

Comment:
Why?

Reply:
this means is an advance template.

Reply:
Got it here, please add a comment that said:

"Right now we only allowing users to edit layout, so no templates or advanced template can be edit from here"